src/material-experimental/mdc-form-field/_form-field-native-select.scss

```diff
@@ -0,0 +1,90 @@
+@import '../../cdk/a11y/a11y';
+
+// Width of the Material Design form-field select arrow.
+$mat-form-field-select-arrow-width: 10px;
+// Height of the Material Design form-field select arrow.
+$mat-form-field-select-arrow-height: 5px;
+// Horizontal padding that needs to be applied to the native select in an form-field so
+// that the absolute positioned arrow does not overlap the select content.
+$mat-form-field-select-horizontal-end-padding: $mat-form-field-select-arrow-width + 5px;
+
+// Mixin that creates styles for native select controls in a form-field.
+@mixin _mat-form-field-native-select() {
+  // Remove the native select down arrow and ensure that the native appearance
+  // does not conflict with the form-field. e.g. Focus indication of the native
+  // select is undesired since we handle focus as part of the form-field.
+  select.mat-mdc-input-element {
```

Comment:
It looks like some of these styles technically probably belong with the MatInput. Is this what we do with the non-MDC version as well?

Reply:
Yes. They belong to `MatInput`, but since that one is a directive, the styles need to be part of the form-field. That's how its done in the non-MDC version too (unfortunately)